services/management/commands/turku_service_import/units.py

```diff
@@ -177,6 +185,38 @@ def _handle_services_and_service_nodes(self, obj, unit_data):
 
         set_syncher_object_field(obj, 'root_service_nodes', ','.join(str(x) for x in obj.get_root_service_nodes()))
 
+    def _handle_service_offers(self, obj, unit_data):
+        descriptions = {
+            'fi': getattr(obj, 'description_fi', ''),
+            'sv': getattr(obj, 'description_en', ''),
+            'en': getattr(obj, 'description_sv', ''),
+        }
+        touched = {
+            'fi': False,
+            'sv': False,
+            'en': False,
+        }
+
+        for service_offer in unit_data.get('palvelutarjoukset', []):
+            offer_name = service_offer.get('nimi_kieliversiot', {})
+            for language, value in offer_name.items():
+                # Make sure that we have a string as the default value
+                if not descriptions[language]:
+                    descriptions[language] = ''
+
+                if not touched[language]:
+                    # Add spacing and header for service offers
+                    descriptions[language] = clean_text(descriptions[language])
+                    descriptions[language] += '\n\n' + SERVICE_OFFERS_TRANSLATIONS[language] + ':\n'
```

Comment:
`clean_text()` can return `None` (which is bad IMO), in which case this row will error.